go.mod

```diff
@@ -4,6 +4,8 @@ require (
 	cloud.google.com/go v0.28.0 // indirect
 	github.com/DataDog/datadog-go v0.0.0-20180822151419-281ae9f2d895 // indirect
 	github.com/abbot/go-http-auth v0.0.0-20150922224136-efc9484eee77
+	github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc // indirect
```

Comment:
is it worth to import all these dependencies for those few assertions in the tests? They slow down the builds.

Reply:
Fixed, not sure where they came from.